src/db.js

```diff
@@ -10,11 +10,44 @@ const createTables = () => {
       performer TEXT NOT NULL,
       content TEXT NOT NULL,
       deadline INTEGER NOT NULL,
+      expiredNotificationSent BOOLEAN NOT NULL DEFAULT 0,
       created_at INTEGER NOT NULL
-    )`
+    );`
   );
 };
 
+const migrateIfNeeded = async () => {
+  // if there is no table schemaVersions, currentVersion is 1
+  let currentVersion = (await db.get(`
```

Comment:
good suggestion, I wanted to do something like this but failed and used a `let`, this is much better